20260205 truncate service acct tokens (#4113)
kanidm/kanidm: book/src/accounts/service_accounts.md

ldapwhoami -H ldaps://idm.example.com -x -D "dn=token" -w "..."
# u: demo_service@idm.example.com
```

### Compact API Tokens

API Tokens issued by Kanidm internally are a JWS object that contains metadata about the service account that is being

> granted access. This means that the token can become quite long to encode the necessary information.

Reviewer: ```suggestion
granted access. To encode the necessary information, the token can be quite long.
```
